components/statewide-header/index.css

```diff
@@ -147,4 +161,20 @@
   display: none;
 }
 
+.mobile-only {
+  display: none;
+}
+
+@media screen and (min-width: 800px) {
+  .desktop-only {
+    display: inline;
+  }
+}
+
+@media screen and (max-width: 799px) {
```

Comment:
Can we skip this if we hide the desktop-only section by default

Reply:
https://codepen.io/alexhu00/pen/wvpmWQE

Like this?

Reply:
```
.desktop-only {
  display: none;
}

@media screen and (min-width: 800px) {
  .desktop-only {
    display: inline;
  }
  .mobile-only {
    display: none;
  }
}
```

Reply:
Yes

Reply:
Love seeing this in action on the codepen. I think if we hide the desktop-only section by default and only show it on wide screens with the media query we can avoid using a mobile-only span at all. Thanks!

Reply:
Got it I see. I wasn't sure how to add an extra space at the end of a span with HTML which is why I at first created two classes but was able to do so with the &nstp;

https://codepen.io/alexhu00/pen/abEjjmK

`        <span class="desktop-only"
          >Official website of the&nbsp;</span
        ><span>State of California </span>
`

`.desktop-only {
  display: none;
}

@media screen and (min-width: 800px) {
  .desktop-only {
    display: inline;
  }
}`

Let me know if this is what you meant.